### PR TITLE
jfrog-cli-go 1.34.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
build failure
```
==> brew install --verbose --build-bottle jfrog-cli-go
==> FAILED
/usr/bin/sandbox-exec -f /private/tmp/homebrew20200303-80131-6ncz24.sb nice ruby -W0 -I $LOAD_PATH -- /usr/local/Homebrew/Library/Homebrew/build.rb /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/jfrog-cli-go.rb --build-bottle --verbose
==> Downloading https://github.com/JFrog/jfrog-cli-go/archive/1.34.0.tar.gz
Already downloaded: /Users/brew/Library/Caches/Homebrew/downloads/b1f54194c5e47f2d43e4edc130aae446f3f47d5c67043b5c65837929fc5f7fb2--jfrog-cli-1.34.0.tar.gz
==> Verifying b1f54194c5e47f2d43e4edc130aae446f3f47d5c67043b5c65837929fc5f7fb2--jfrog-cli-1.34.0.tar.gz checksum
tar xof /Users/brew/Library/Caches/Homebrew/downloads/b1f54194c5e47f2d43e4edc130aae446f3f47d5c67043b5c65837929fc5f7fb2--jfrog-cli-1.34.0.tar.gz -C /private/tmp/d20200303-80132-1ni9tcg
cp -pR /private/tmp/d20200303-80132-1ni9tcg/jfrog-cli-1.34.0/. /private/tmp/jfrog-cli-go-20200303-80132-1f1fxcu/jfrog-cli-1.34.0
chmod -Rf +w /private/tmp/d20200303-80132-1ni9tcg
==> go run ./python/addresources.go
stat ./python/addresources.go: no such file or directory
```

**Comment by Bo98** [comment]
I think you can just delete the line `system "go", "run", "./python/addresources.go"`. Seems to have been removed with no replacement.

**Comment by chenrui333** [comment]
@Bo98 yeah, that works. Gonna to search how the build got modified in the upstream codebase.

File: Formula/jfrog-cli-go.rb [hunk 1]
```diff
@@ -14,7 +14,6 @@ class JfrogCliGo < Formula
   depends_on "go" => :build
 
   def install
-    system "go", "run", "./python/addresources.go"
```
Comment (chenrui333) [inline Formula/jfrog-cli-go.rb]: This step got removed in this commit, https://github.com/jfrog/jfrog-cli/commit/94dd4c1f4287a1c5b645f332dd235a83001f4acf#diff-0b83f9dedf40d7356e5ca147a077acb4

**Pull request closed without merging**